readthedocs_theme/templates/includes/topnav.html

```diff
@@ -23,7 +23,7 @@
     <i class="fad fa-browser primary icon"></i>
     Hosting features
   </a>
-  <a class="item" href="{{ SITEURL }}/features/#reading">
+  <a class="item" href="{{ SITEURL }}/features/reader/">
```

Comment:
If we want to do that, we should probably move the page to `/features/authoring/`? But not sure if they're doing the same thing or not.

Reply:
I'm suggesting that because the link from

![Screenshot_2024-04-25_10-26-08](https://github.com/readthedocs/website/assets/244656/9aa91459-ceb5-4962-98d2-26f56d8cd03f)

points to https://about.readthedocs.com/features/#authoring, which also says "Maintain docs as code", and that is exactly what the page Docs As Code is about:

![Screenshot_2024-04-25_10-27-03](https://github.com/readthedocs/website/assets/244656/8be54e9f-5783-403a-a1d2-a9fea5482b90)

Reply:
Cool, I will link it for now, but think we probably want to rename this file to `/features/authoring/` if thats our plan.